common/src/main/java/whocraft/tardis_refined/common/crafting/astral_manipulator/ManipulatorCraftingRecipe.java

```diff
@@ -127,6 +127,14 @@ public boolean canCraftInDimensions(int width, int height) {
 
     @Override
     public ItemStack getResultItem(RegistryAccess registryAccess) {
+        if(result.type() instanceof ManipulatorItemResult manipulatorItemResult){
+            return manipulatorItemResult.recipeOutput();
```

Comment:
Why is this needed? I thought this only get used in vanilla crafting benches.

Reply:
You're correct, but if for some weird reason a mod tries to render our recipes out, its better to render something than nothing

Reply:
Good catch, nice one.